src/debug/nodejsDebugProvider.ts

```diff
@@ -103,4 +118,15 @@ export class NodejsDebugProvider implements IDebugProvider {
     public isPortRequired(): boolean {
         return true;
     }
+
+    public async getDebugArgs(): Promise<string | undefined> {
+        const debugCommand = await vscode.window.showInputBox({
+            prompt: 'Debug command for your container:',
```

Comment:
Not quite sure what "Debug command for your container" means... but not sure what a better prompt would be either!

Reply:
I used this as it was already used in the other debug command and because i didn't find anything better .... but i agree with your concern. I changed it. Let me know if you like it more.